hnn_core/tests/test_network.py

```diff
@@ -350,19 +348,22 @@ def test_tonic_biases():
                        ' negative'):
         net.add_tonic_bias(cell_type='L2_pyramidal', amplitude=1.0,
                            t0=5.0, tstop=4.0)
+        simulate_dipole(net, tstop=5.)
+    net.external_biases = dict()
```

Comment:
Why not just call [`clear_drives()`](https://github.com/jonescompneurolab/hnn-core/blob/78ce7cf73607135475a19718c79fa41304aaff80/hnn_core/network.py#L1106)?

Also, I feel like the this should set the duration to zero, not negative. It just might seem confusing to a user given that they may have entered a perfectly valid start and end time that happens to fall outside the bounds of the simulation.

Reply:
sorry if this was confusing. The `tstop` in `simulate_dipole` was arbitrary. I just needed `simulate_dipole` to invoke the error because the checks now happen in that function. 

I'll change it to `clear_drives`

Reply:
actually `clear_drives` will not work in this context because we are dealing with tonic bias which is not considered a drive?

Reply:
Oh shoot, you're right. For some reason I though `clear_drives` also cleared biases. Do we have a proper way to clear biases or this it?

Reply:
I guess we could add a method but I find this simple enough

Reply:
We can address this in a later PR if it comes up again.